.github/workflows/sync-triage-config.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - master
 
+permissions:
+  contents: read
```

Comment:
Does `peter-evans/create-pull-request` require additional scopes? The [documentation](https://github.com/peter-evans/create-pull-request#action-inputs) says:

> `GITHUB_TOKEN` (`contents: write`, `pull-requests: write`) or a `repo` scoped [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).

I've never really had a good understanding of how the tokens work so It's possible this is indeed enough

Reply:
We're not currently using `GITHUB_TOKEN` for that:

```
token: ${{ secrets.HOMEBREW_DOTGITHUB_WORKFLOW_TOKEN }}
```